src/client-apps/nuxeo-drive/nuxeo-drive-faq/nuxeo-drive-architecture-recommendations-windows.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Nuxeo Drive Configuration Requirements on Windows
+review:
+  comment: ''
+  date: '2024-10-05'
+  status: ok
+labels:
+  - nuxeo-drive
+  - mlumeau
+  - multiexcerpt
+  - excerpt
+  - lts2023-ok
+toc: true
+tree_item_index: 100
+---
+
+## Before You Install
+
+### Supported Windows Version 
+
+- Windows >= 10, both 32 and 64 bits
+
+### Nuxeo Drive Version 
+
+All Nuxeo Drive versions are compatible with Windows. We recommend to use the latest stable version to benefit from the latest bugfixes and improvements.
+  
+### CPU and RAM Requirements 
+
+Resources consumption depends mostly on the third-party application used for file editing as well as the size of the file. 
+### Requirements for Direct Edit
+Nuxeo Drive is tested against Microsoft Office 365 and is therefore our recommendation. Direct edit can also be used with alternative Office suites, but has not been tested against them.
+
+To use Direct Edit, you should allocate at least **500MB of free memory** for Nuxeo Drive on top of Microsoft Office consumption for direct edit usage or at least **twice the file size** for larger files.
+
+### Permissions 
+
+Administrator rights are required on the machine while installing or uninstalling the application. Upgrade is similar to installation, so a similar level of access is required.  
+
+### Dependencies 
+
+Starting from Nuxeo-drive version 5.5.0 => Python v3.9.5 is compatible and required to be installed separately in the local machine. 
```

Comment:
Python 3.9.5 is supported from 5.2.0
[NXDRIVE-2629](https://jira.nuxeo.com/browse/NXDRIVE-2629): Upgrade from Python 3.9.4 to 3.9.5